treemap/script.js

```diff
@@ -122,90 +115,89 @@ d3.json('data.json', function(err, dataset) {
     });
 
   // 💩 Manual labelling
-  var key = [
+  const key = [
     { name: 'Europe', color: '#254251' },
     { name: 'Britain', color: '#E0AB26' },
     { name: 'Africa', color: '#3292A6' },
     { name: 'S. America', color: '#F37F2F' },
   ];
 
-  var legendx = config.width < 400 ? 10 : width + 10;
-  var legendy = config.width < 400 ? 310 : 10;
-  var legendheight = config.width < 400 ? 10 : 20;
+  const legendx = width + 10;
+  const legendy = 10;
+  const legendheight = 20;
```

Comment:
Can we camelCase these variable names? Or otherwise create a `legend = { x, y, height }` object?